src/tokens/fren.ts

```diff
@@ -0,0 +1,26 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+import { getAmountInAddresses, getBlockFrostInstance } from "../utils";
+
+const fren =
+  "fc11a9ef431f81b837736be5f53e4da29b9469c983d07f321262ce614652454e";
+
+const frenFetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+  const blockFrost = getBlockFrostInstance(options);
+  const total = 420_069_000_000n;
```

Comment:
change `const total = 420_069_000_000n;` -> `const total = 420_069_000_000;`

test fails due to different types (`TypeError: Cannot mix BigInt and other types, use explicit conversions`)

Reply:
Thanks for the help.
What do I need to add in the index.ts in the dictionary<string, fetcher>?

???????????:frenFetcher